packages/aila/src/core/Aila.ts

```diff
@@ -242,7 +239,7 @@ export class Aila implements AilaServices {
   }
 
   public get rag() {
-    return this._rag;
+    throw new Error("Attempting to use old, unused RAG service");
```

Comment:
The RAG service seems to be unused! So, removing it so that it's clearer what is doing what. RAG seems to all happen within the prompt builder now